src/state-summary/state-card-analog_output.ts

```diff
@@ -0,0 +1,197 @@
+import "@polymer/iron-flex-layout/iron-flex-layout-classes";
+import { IronResizableBehavior } from "@polymer/iron-resizable-behavior/iron-resizable-behavior";
+import "@polymer/paper-input/paper-input";
+import { mixinBehaviors } from "@polymer/polymer/lib/legacy/class";
+import { html } from "@polymer/polymer/lib/utils/html-tag";
+import { PolymerElement } from "@polymer/polymer/polymer-element";
+
+import "../components/entity/ha-entity-toggle";
+import "../components/entity/state-info";
+import "../components/ha-slider";
+
+class StateCardAnalogOutput extends mixinBehaviors(
+  [IronResizableBehavior],
+  PolymerElement
```

Comment:
You can't use Polymer for new elements, please use `LitElement`

Reply:
Hi Bram, I replaced PolymerElement with LitElement, but the slider, the desciption, the state and the toggle button disappear. I copied this code from state-card-input_number.js, and modified to my needs. I checked the other state-card-*.js files, they al seem to use this polymerelement. But as you mentioned before, you want to have ts-files. Is there an example how to create a slider-interface with a ts file that I can use as an example?

Reply:
https://github.com/home-assistant/frontend/blob/dev/src/panels/lovelace/entity-rows/hui-input-number-entity-row.ts#L81

Reply:
Hm, this is for a row entity, that's something different from the state-summary. Actually I have one example in the state-summaries, that's state-card-input_select.ts. And that is quite different as it uses the InputSelectEntity from types to create a stateObj that can be used to extract the data. But my object is not an InputSelect, it's a slider. I am trying to combine the different examples, but till now I don't solve the puzzle. What object is the best to create my own stateObj? Should I use the InputSelectEntity as well?

Reply:
Done. Better ts file is pushed.